sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/MessageLockLostEventArgs.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Messaging.ServiceBus
+{
+    /// <summary>
+    /// This type represents the event args relating to the message lock lost event.
+    /// </summary>
+    public class MessageLockLostEventArgs
```

Comment:
Are we able to include the lock token in here?  I know that we're exposing this from the args for a single message being processed, but there's a chance that callers are going to want to call one function that does the work, such as keeping a dictionary of lock token-to-cancellation token associations.

Reply:
Good catch - we can probably just include the whole ServiceBusReceivedMessage in case they want to access any of the properties.